pkgs/top-level/all-packages.nix

```diff
@@ -11763,7 +11763,7 @@ with pkgs;
 
   darwin = let
     apple-source-releases = callPackage ../os-specific/darwin/apple-source-releases { };
-  in apple-source-releases // rec {
+  in recurseIntoAttrs (apple-source-releases // rec {
```

Comment:
We could explicitly disable those with `cf-private = {};` in the release.nix, apart from that it should be ok.